Source/api/Utils/PartUtilsImpl.cs

```diff
@@ -113,7 +113,7 @@ public class PartUtilsImpl {
   /// The root game object of the new hirerarchy. This object must be explicitly disposed when not
   /// needed anymore.
   /// </returns>
-  public GameObject GetSceneAssemblyModel(Part rootPart, bool goThruChildren = true) {
+  public GameObject GetSceneAssemblyModel(Part rootPart, bool goThruChildren = true, bool keepColliders = false) {
```

Comment:
Even though the name of the parameter is self-explanatory, could you please modify the doc too? It helps running doc generation tools.

Reply:
Ah, sorry, I forgot to do so.